TurkishFilter/sections/specific.txt

```diff
@@ -4163,6 +4163,7 @@ r10.net##table[id^="post"] + div[style^="padding: 6px 0px 0px"]
 erzurumgazetesi.com.tr##table[width="468"]
 isguc.org##td[width="200"] a[target="_blank"] img[width="200"][alt]
 guncelgazete.com##td[width="215"][valign="top"] > table[width="200"][border="0"]
+donanimhaber.com##.miktar:contains(/reklam/i):upward(3)
```

Comment:
Your rule is good, but you can do it without `:contains`.
Add the rule below to the same filter, but to `css_extended.txt`.
```
donanimhaber.com#?#nav[id^="NavMenu"] > ul.oge:has(> li > a[href^="https://disneyplus.bn5x.net/"])
```

Reply:
Next time please fill out of the pull request more properly (with screenshots, the name of the task in the description).

Reply:
Can you show other examples? For now I can see only the Disney+ ad.

Reply:
If there are other ads, you can add this rule with `:contains`.
Regex is not required.
```
donanimhaber.com#?#nav[id^="NavMenu"] > ul.oge:has(> li > a[href] > span.hot-topic-word > span.miktar:contains(Reklam))
```

Reply:
@c0b41 please do next:
1. remove `donanimhaber.com##.miktar:contains(/reklam/i):upward(3)`
2. exchange `donanimhaber.com#?#nav[id^="NavMenu"] > ul.oge:has(> li > a[href^="https://disneyplus.bn5x.net/"])` to `donanimhaber.com#?#nav[id^="NavMenu"] > ul.oge:has(> li > a[href] > span.hot-topic-word > span.miktar:contains(Reklam))`